pkg/auth/authLogout_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
```

Comment:
I'm wondering whether we should use `auth logon` and `auth logoff` instead of `auth login` and `auth logout` ? What does @CaroMac think ?

Reply:
I prefer auth login and auth logout - when I think of more open source type sites I think this term is more commonly used. Also login suggests that you need to provide credentials, whereas you can logon to sites without necessarily needing to provide credentials.

Reply:
OK